queries.sql

```diff
@@ -13,3 +13,80 @@ SELECT * FROM animals WHERE neutered IS TRUE;
 SELECT * FROM animals WHERE name != 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+/* update the animals table by setting the species column to unspecified */
+-- Begin Transaction
+BEGIN; 
+
+-- Update species column to 'unspecified'
+UPDATE animals
+SET species = 'unspecified';
+
+-- verify that change was made
+SELECT * from animals;
+
+-- roll back and verify that species column went back to state
+ROLLBACK;
+
+-- verify changes
+SELECT * from animals; 
+
+-- Update species column based on a condition
+-- Transaction begins
+  BEGIN;
+
+  -- update species column for animals with names ending in mon
+  UPDATE animals
+  SET species = 'digimon'
+  WHERE name LIKE '%mon';
+
+  -- update species column for animals with names not ending in mon
+  UPDATE animals
+  SET species = 'pokemon'
+  WHERE name NOT LIKE '%mon';
+
+  -- commit transaction
+  COMMIT;
+
+  -- verify changes
+  SELECT * from animals
+
+-- Delete records
+DELETE from animals;
+
+-- verify all records are deleted
+SELECT * from animals;
+
+-- rollback changes
+ROLLBACK;
+
+-- verify all records still exist
+SELECT * from animals;
+
+-- queries
+-- begin transaction
+BEGIN;
+
+-- delete all animals born after Jan 1st 2022
+DELETE from animal
+WHERE date_of_birth > '01-01-2022';
+
+-- create a savepoint for the transaction
+SAVEPOINT SP1;
+
+-- update all animals' weights that are negative multiplied by -1
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+
+-- commit the transactions
+COMMIT;
+
+
+
+
+
+
+
+
```

Comment:
- [ ] Good job on the queries, but please kindly include queries to answer the below questions
- How many animals are there?
- How many animals have never tried to escape?
- What is the average weight of animals?
- Who escapes the most, neutered or not neutered animals?
- What is the minimum and maximum weight of each type of animal?
- What is the average number of escape attempts per animal type of those born between 1990 and 2000?